diff --git a/setup.py b/setup.py
index 86dd437..ba313d8 100644
--- a/setup.py
+++ b/setup.py
@@ -23,11 +23,12 @@ setup(name="cdtime",
       packages=['cdtime'],
       package_dir={'cdtime': 'Lib'},
       include_dirs=['Include', 'Include/py3c'] + cdat_info.cdunif_include_directories,
-      ext_modules=[Extension('cdtime._cdtime',
-                             ['Src/cdtimemodule.c'],
-                             library_dirs=cdat_info.cdunif_library_directories,
-                             libraries=cdat_info.cdunif_libraries,
-                             runtime_library_dirs=[libs_pth],
-                             extra_compile_args=["-L%s" % libs_pth, "-g", "-O0"],)
-                   ]
-      )
+      ext_modules=[
+          Extension(
+              'cdtime._cdtime',
+              ['Src/cdtimemodule.c'],
+              include_dirs=[sys.prefix+'/include'],
+              library_dirs=[sys.prefix+'/lib'],
+              libraries=['cdms', 'netcdf', 'g2c', 'png', 'jasper'],
+        )
+      ])
